css/style.css

```diff
@@ -0,0 +1,29 @@
+body {
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 15px;
+    background-color: rgb(133, 163, 133);
+}
```

Comment:
You have done a great job, but i suggest you some minor changes which you may follow:
In the body selector, the background color is specified as rgb(133, 163, 133). It seems there might be a typo, as the same value is used for the red and blue channels. If you intended to use a shade of green, you can modify it to rgb(133, 163, 193) or any other appropriate value.

Reply:
- [ ] This is not a valid review, is your opinion based on a design ? Please look through the code and catch errors or areas of improvement. refer to this [document](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md) 👍🏾